.gitignore

```diff
@@ -30,4 +30,8 @@ ipch/
 *.suo
 *.user
 *.userosscache
-*.sln.docstates
\ No newline at end of file
+*.sln.docstates
+
+# Build results
+[Dd]ebug/
+[Rr]elease/
```

Comment:
Is it possible to change the solution to build into `build/vs`? I have the XCode project on Mac building into `build/xcode`. This ensures build outputs don't clutter up the top level for every platform.